validphys2/src/validphys/pseudodata.py

```diff
@@ -369,15 +369,21 @@ def make_level1_data(data, level0_commondata_wc, filterseed, experiments_index,
 
     indexed_level1_data = indexed_make_replica(experiments_index, level1_data)
 
+    dataset_order = [cd.setname for cd in level0_commondata_wc]
```

Comment:
not that it matters much, but we should be doing this with a dict rather than a list, to avoid quadratic behavior.